CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-# dbt_quickbooks version.version
+# dbt_quickbooks v0.18.0
+This release introduces the following updates.
+
+## Bug Fixes
```

Comment:
I would mark this as breaking changes since it's called out at the end of the section.

Reply:
Ah good catch!